doc/man3/RAND_add.pod

```diff
@@ -54,6 +57,12 @@ should consider using L<RAND_load_file(3)> instead.
 
 RAND_seed() is equivalent to RAND_add() with B<randomness> set to B<num>.
 
+RAND_keep_random_devices_open() is used to control file descriptor use by
+the random seed sources. Some seed sources maintain open file descriptors
+by default. This allows such sources to operate in a chroot(2) jail
```

Comment:
s/by default. This allows/by default, which allows/

Reply:
Done.